packages/fast-components-styles-msft/src/context-menu-item/index.ts

```diff
@@ -0,0 +1,70 @@
+import designSystemDefaults, {
+    DesignSystem,
+    withDesignSystemDefaults,
+} from "../design-system";
+import { ComponentStyles } from "@microsoft/fast-jss-manager";
+import { ContextMenuItemClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+import { density } from "../utilities/density";
+import { defaultHeight, maxHeight, minHeight } from "../utilities/height";
+import {
+    backgroundColor,
+    disabledContrast,
+    ensureForegroundNormal,
+    hoverContrast,
+} from "../utilities/colors";
+import { contrast, scaleContrast } from "@microsoft/fast-jss-utilities";
+import { applyType, TypeRamp } from "../utilities/typography";
+
+const styles: ComponentStyles<ContextMenuItemClassNameContract, DesignSystem> = {
+    contextMenuItem: {
+        listStyleType: "none",
+        height: density(40),
+        display: "grid",
+        gridTemplateColumns: "40px auto 40px",
+        gridTemplateRows: "auto",
+        alignItems: "center",
+        padding: "0",
+        color: ensureForegroundNormal,
+        whiteSpace: "nowrap",
+        overflow: "hidden",
+        cursor: "default",
+        ...applyType("t7", "vp1"),
+        background: backgroundColor,
+        "&:focus": {
+            outline: "none",
+        },
+        "&:focus, &:hover": {
+            background: (config: DesignSystem): string => {
+                const designSystem: DesignSystem = withDesignSystemDefaults(config);

```

Comment:
nit: why not have `const designSystem: DesignSystem = withDesignSystemDefaults(config);` at the top since you re-declare this more than once?

Reply:
Because this stylesheet isn't a function. There are benefits to the stylesheet being an object and not a function, so in general this approach is preferred.